source/dsl/cli-args.ts

```diff
@@ -15,6 +15,8 @@ export interface CliArgs {
   dangerfile: string
   /** So you can have many danger runs in one code review */
   id: string
+  /** Use staged changes */
+  staged?: boolean
```

Comment:
Are you sure this works? The option above is `  .option("-s, --staging", "Just use staged changes.")` - so I'd expect this to be `staging`?

Reply:
In my local test. It works well. I will update the pull request later.